modules/cij/runner.py

```diff
@@ -790,8 +822,8 @@ def main(conf):
             tsuite.enter(trun)
 
             for tcase in (tc for tc in tsuite.testcases if tsuite.entered):
-                tcase_match = conf.get("TESTCASE_MATCH", None)
-                if tcase_match is None or tcase_match in tcase.name:
+                if args.testcase_match is None or \
+                        args.testcase_match in tcase.name:
```

Comment:
Could be written using parenthesis instead of the line continuation `\`

Reply:
Nice! Fixed.